atomata/capstone_WebClient
# Azure Json Connections For Loading Apparatuses

### Summary 
Json files for apparatuses are now obtained through the Azure DB
### Details
-Only affects retrieving jsons from Azure
-Note that Apparatus List on start isn't connected yet
### How to test?
Literally just run it and look to see if anything loads. If not then there's a problem.
